packages/amplify-provider-awscloudformation/src/aws-utils/aws-cfn.js

```diff
@@ -335,7 +335,7 @@ class CloudFormation {
     do {
       const listStackResourcesParams = {
         StackName: parentStackName,
-        nextToken,
+        NextToken: nextToken,
```

Comment:
you are also going to want to double check that it supports undefined as a value in here for the call.

Reply:
Yup, using an already existing util function that we have to do this which takes care of these checks.